doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -555,6 +551,21 @@ Bluetooth samples
 
   * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
 
+Bluetooth Fast Pair samples
+---------------------------
+
+* Defined the new :ref:`bt_fast_pair_samples` category and moved the :ref:`fast_pair_input_device` sample into it.
```

Comment:
This could be moved to sample sections - https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-15961/nrf/releases_and_maturity/releases/release-notes-2.7.0.html#samples. Just the  category part.
```suggestion
* Moved the :ref:`fast_pair_input_device` sample to the :ref:`bt_fast_pair_samples` category.
```